include/mbedtls/platform.h

```diff
@@ -163,11 +164,11 @@ int mbedtls_platform_set_fprintf( int (*fprintf_func)( FILE *stream, const char
 extern int (*mbedtls_printf)( const char *format, ... );
 
 /**
- * \brief   Set your own printf function pointer
+ * \brief    This function sets your own printf function pointer.
```

Comment:
I don't like our wording "your own xxx function pointer" very much here, and while we're at it, we might consider improving it as well. What about "This function allows to configure a custom `fprintf` function pointer." or "This function sets a custom `fprintf` function pointer." - @Patater ?

Reply:
I agree. I remarked on this to Ron yesterday. I changed to your first suggestion for all functions.